lib/cucumber/formatter/pretty.rb

```diff
@@ -264,7 +264,7 @@ def print_comments(up_to_line, indent)
         return if comments.empty? || comments.length <= @next_comment_to_be_printed
         comments[@next_comment_to_be_printed..-1].each do |comment|
           if comment.location.line <= up_to_line
-            @io.puts(format_string(comment.text.strip, :comment).indent(indent))
+            @io.puts(indent(format_string(comment.text.strip, :comment), indent))
```

Comment:
@aurelien-reeves ie here I would have the method arg 2 renamed to `indent_amount` or `padding` if you like

Reply:
Oh, ok, got it 👍